RetailCoder.VBE/Inspections/VariableRequiresSetAssignmentEvaluator.cs

```diff
@@ -0,0 +1,160 @@
+﻿using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Inspections
+{
+    class VariableRequiresSetAssignmentEvaluator
```

Comment:
@retailcoder - Yeah, probably.  I'm thinking that's a big chunk of tech debt that needs to be paid eventually.  At this point it makes more sense to keep functionality consolidated there though, because when it comes time to pivot to a more tree-like structure it will make it easier to do the refactoring.